src/custom/components/ErrorBoundary/ErrorBoundaryMod.tsx

```diff
@@ -204,47 +206,53 @@ export default class ErrorBoundary extends React.Component<unknown, ErrorBoundar
             </HeaderRow>
           </HeaderWrapper>
           <Wrapper>
-            <FlexContainer>
-              <StyledTitle>
-                <Trans> Something went wrong</Trans>
-              </StyledTitle>
-              <img src={CowError} alt="CowSwap Error" height="125" />
-            </FlexContainer>
-            <AutoColumn gap={'md'}>
-              <CodeBlockWrapper>
-                <code>
-                  <ThemedText.Main fontSize={10} color={'text1'}>
-                    <StyledParagraph>{error.stack}</StyledParagraph>
-                  </ThemedText.Main>
-                </code>
-              </CodeBlockWrapper>
-              <AutoRow>
-                <LinkWrapper>
-                  <ExternalLink
-                    id="create-github-issue-link"
-                    href={
-                      CODE_LINK +
-                      `/issues/new?assignees=&labels=🐞 Bug,🔥 Critical&body=${encodedBody}&title=${encodeURIComponent(
-                        `Crash report: \`${error.name}${error.message && `: ${truncate(error.message)}`}\``
-                      )}`
-                    }
-                  >
-                    <ThemedText.Link fontSize={16}>
-                      <Trans>Create an issue on GitHub</Trans>
-                      <span>↗</span>
-                    </ThemedText.Link>
-                  </ExternalLink>
-                </LinkWrapper>
-                <LinkWrapper>
-                  <ExternalLink id="get-support-on-discord" href={DISCORD_LINK}>
-                    <ThemedText.Link fontSize={16}>
-                      <Trans>Get support on Discord</Trans>
-                      <span>↗</span>
-                    </ThemedText.Link>
-                  </ExternalLink>
-                </LinkWrapper>
-              </AutoRow>
-            </AutoColumn>
+            {error?.name === 'ChunkLoadError' ? (
+              <ChunkLoadError />
+            ) : (
+              <>
+                <FlexContainer>
```

Comment:
If you want, because this component has been modified so much from the original, we can start to own this component. 

It doesn't compare any more 1<>1, so then I would just would make some component for the `else` part of this code, just for readability.

```
{error?.name === 'ChunkLoadError' ? (
    <ChunkLoadError />
  ) : (
      <InternalError />
)}